README.md

```diff
@@ -19,6 +19,41 @@ The markdown should be rendered to HTML before sending; the VSCode markdown prev
 
 ## How to send emails
 
+### Within a competition cycle
+
+During a competition cycle the emails can be sent in one of two ways.
+Since we offer teams the option of having their secondary contact(s) CCd into emails, we are unable to use GMail's mail merge feature.
+
+For either of these routes you will need to have configured your SR GMail account to be able to send from `teams@`.
+
+#### Personalised emails
+
+1. Set up a Python virtual environment and `pip install -r scripts/requirements.txt`
+1. Export as a CSV the "Combined" tab of the current year's "Teams Organisation (internal)" spreadsheet
+1. Pre-process that file using `./scripts/process-export.py $EXPORTED.csv > $EXPORTED.processed.csv`
+1. Send the email using `./scripts/send.py`:
+    1. Create a fresh "app password" for your account for use by the script; set that via ` export SMTP_PASSWORD=...`
+    1. Set your username via `export SMTP_USERNAME=...`
+    1. Dry-run the send via the `--dry-run`, check that the printed output is what you expect
+    1. You may also want to dummy send the email to yourself; this requires creating a dummy CSV with equivalent columns to the processed file and then sending the email. Be sure you're sending to the right file when doing this!
+    1. Send the actual email. For 30 teams this takes a couple of minutes as the script rate-limits itself to avoid hitting GMail sending limits.
+    1. Delete from your account the app password you created in the earlier step
+
+#### Mailshots
+
+When no personalisation is needed, the email can be sent as a mailshot.
+This involves:
+
+* Rendering the email markdown yourself (VSCode's preview tab does this well for example)
```

Comment:
Question: any reason not to use GitHub's render? If someone doesn't use VSCode, our emails will render inconsistently

Reply:
I can't recall if there's a reason not to use the GitHub one, though the VSCode one is what's mostly been used (i.e: I've used it exclusively and I've sent most of this year's emails). As the personalised ones are rendered differently anyway, the incremental value of consistency here is small. Better might be to have CI render all emails using the same renderer as the personalised emails get, though we haven't been doing that.

Reply:
Oh, also just spotted that the VSCode preview renderer is already what's suggested elsewhere in the README too.